argo-lite/src/graph-frontend/src/api.js

```diff
@@ -110,8 +110,8 @@ module.exports = function(self) {
     // This is because CSSRenderer is slow.
     if (nodes.length == 0) {
       self.turnOffLabelCSSRenderer();
-    } else {
-      // TODO: Only turn on when no node is moving?
+    } else if (appState.graph.frame.paused) {
```

Comment:
This won't work when users are dragging node. This TODO is actually not as easy to implement and might need some more advanced solution.